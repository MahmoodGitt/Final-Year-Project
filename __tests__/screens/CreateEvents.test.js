/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';

import SignOut from '../../screens/CreateEvent';

describe('CreateEvent Component', () => {
	it('Testing if Sign out Component loads up', () => {
		const tree = renderer.create(<SignOut />).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
