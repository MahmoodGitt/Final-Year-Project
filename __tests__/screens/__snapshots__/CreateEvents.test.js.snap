// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateEvent Component Testing if CreateEvent Component loads up 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
/>
`;

exports[`CreateEvent Component Testing if Sign out Component loads up 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
/>
`;
